src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,72 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+
+        const startAnimation = () => {
+          if (ripple) return;
+          ripple = document.createElement("mwc-ripple");
+          this.shadowRoot.querySelector("div").appendChild(ripple);
+          ripple.updateComplete.then(() => {
```

Comment:
Why wait to updateComplete? that means it will render twice? Also, instead of attributes you can set properties: `ripple.unbounded = true`

Reply:
also, you could use 
```js
async () => {
  await ripple.updateComplete;
  ripple.setAttribute(…)
}

Reply:
Setting ripple active doesn't work until it has rendered once.